sdk/src/main/java/io/dapr/client/DaprClient.java

```diff
@@ -662,6 +665,15 @@ Flux<SubscribeConfigurationResponse> subscribeConfiguration(String storeName, Li
    * @return Mono of {@link UnsubscribeConfigurationResponse} instance.
    */
   Mono<UnsubscribeConfigurationResponse> unsubscribeConfiguration(UnsubscribeConfigurationRequest request);
+  
+  /**
+   * ScheduleJobAlpha1 creates and schedules a job.
+   * 
+   * @param <T> The type of the data for the job.
+   * @param job job to be scheduled
+   * @return a Mono plan of type Void.
+   */
```

Comment:
@charan2628 since this is still early days, we should move it to `DaprPreviewClient` where we host all the "experimental" APIs.

Reply:
Thanks for catching this @artur-ciocanu